pages/linux/procs.md

```diff
@@ -0,0 +1,25 @@
+# procs
+
+> Displays information about the active processes.
+> It can be a used as ps or top alternative.
+> More information: <https://github.com/dalance/procs>.
+
+- List all processes with PID, user, CPU usage, memory usage and the command which started them:
+
+`procs`
+
+- Show information about process with command or user containg `zsh`:
+
+`procs {{zsh}}`
+
+- Show information about all processes sorted by CPU time in [d]escending or [a]scending order:
+
+`procs --sort{{d}}|--sort{{a}} cpu`
+
+- Show information about processes with PID `41` or (command or user) containing (`zsh` `or` `firefox`):
```

Comment:
```suggestion
- Show information about processes with either a PID, command or, user containing (`zsh` `or` `firefox`):
```

Reply:
The comma should go before the `or`.

Reply:
:scream: whoops

Reply:
Why the second `or` has back ticks?

Reply:
Good question. I don't think we need them here.